components/director/internal/domain/bundle/entity.go

```diff
@@ -31,7 +32,13 @@ type Entity struct {
 
 // GetParent returns the parent type and the parent ID of the entity.
 func (e *Entity) GetParent(_ resource.Type) (resource.Type, string) {
-	return resource.Application, e.ApplicationID
+	if e.ApplicationID.String != "" {
+		return resource.Application, e.ApplicationID.String
+	} else if e.ApplicationTemplateVersionID.Valid {
+		return resource.ApplicationTemplateVersion, e.ApplicationTemplateVersionID.String
```

Comment:
this line is not tested by unit tests

Reply:
Same as https://github.com/kyma-incubator/compass/pull/3088#discussion_r1236292893